packages/manager/modules/vps/src/dashboard/migrate/migrate.constants.js

```diff
@@ -0,0 +1,6 @@
+export const MIGRATE_FAQ_LINK =
+  'https://www.ovhcloud.com/lang/vps/vps-offer-migration/';
```

Comment:
```suggestion
  'https://www.ovhcloud.com/lang/vps/vps-offer-migration/';
```
/lang ? Shouldn't we replace for each subsidiary ?